packages/react-components/src/templates/TeamProfileHeader.tsx

```diff
@@ -109,9 +111,11 @@ const TeamProfileHeader: React.FC<TeamProfileHeaderProps> = ({
               ))}
             <li css={extraUsersStyles}>
               {members.length > MAX_MEMBER_AVATARS && (
-                <Avatar
-                  placeholder={`+${members.length - MAX_MEMBER_AVATARS}`}
-                />
+                <Anchor href={`${route.about({}).$}#TEAM_MEMBERS_SECTION`}>
```

Comment:
This hardcoding has two problems:
1. Easy to break by changing strings to be inconsistent
2. No guaranteed uniqueness e.g. when component is mounted multiple times

To solve these, look at how `GroupProfile` in the FE generates a unique id and passes it to the link and down to the element. The lowest common ancestor of the two usages can do the same here (it's probably even in `react-components`, TeamProfilePage or something).

Also, see `GroupProfile.test` for a clean way to test at the common ancestor that the impl avoids the two problems